xc7/make/vivado.cmake

```diff
@@ -1,3 +1,98 @@
+function(COMMON_VIVADO_TARGETS NAME WORK_DIR DEPS BITSTREAM)
+  get_target_property_required(PYTHON3 env PYTHON3)
+  get_target_property(PYTHON3_TARGET env PYTHON3_TARGET)
+
+  add_custom_command(
+    OUTPUT
+        ${WORK_DIR}/design_${NAME}.dcp
+        ${WORK_DIR}/design_${NAME}.xpr
+        ${WORK_DIR}/design_${NAME}.bit
+    COMMAND ${CMAKE_COMMAND} -E remove -f ${WORK_DIR}/design_${NAME}.dcp
+    COMMAND ${CMAKE_COMMAND} -E remove -f ${WORK_DIR}/design_${NAME}.xpr
+    COMMAND ${PRJXRAY_DIR}/utils/vivado.sh -mode batch -source
+        ${CMAKE_CURRENT_BINARY_DIR}/${NAME}_runme.tcl
+        > ${CMAKE_CURRENT_BINARY_DIR}/${WORK_DIR}/vivado.stdout.log
+    WORKING_DIRECTORY ${WORK_DIR}
+    DEPENDS ${DEPS} ${NAME}_runme.tcl
+    )
+
+  add_custom_target(
+      ${NAME}_load_dcp
```

Comment:
`load`? Maybe `show` or `view` would be a better name?

Reply:
FYI - I also logged https://github.com/SymbiFlow/symbiflow-arch-defs/issues/1001